docs/hsi.md

```diff
@@ -512,14 +574,16 @@ References:
 
 Control enforcement technology is available on new Intel platforms and prevents exploits from hijacking the control-flow transfer instructions for both forward-edge (indirect call/jmp) and back-edge transfer (ret).
 
-Possible results:
+**Impact:** A local or physical attacker with an existing unrelated vulnerability can use a ROP gadget to run arbitrary code.
```

Comment:
This is right but layman won't know what it means.

Reply:
`A local or physical attacker with an existing unrelated vulnerability can use a reliable and well-known method to run arbitrary code.` ?

Reply:
That's fine

Reply:
Fixed.